release-please-config.json

```diff
@@ -6,9 +6,9 @@
       ".github",
       ".vscode"
     ],
-    "release-type": "python",
+    "release-type": "simple",
     "include-component-in-tag": false,
-    "include-v-in-tag": true,
+    "include-v-in-tag": false,
```

Comment:
We've been adding a v in the tag since the beginning. Why the change?

Reply:
This was an unneccesary change, intended to change the release-type and the hash to the last release commit